androidApp/build.gradle.kts

```diff
@@ -14,6 +14,14 @@ android {
         versionName = "1.0"
     }
 
+    signingConfigs {
+        create("confetti") {
+            keyAlias = "confetti"
+            keyPassword = "confetti"
+            storeFile = file("confetti.keystore")
+            storePassword = "confetti"
```

Comment:
Putting passwords in clear means anyone can impersonate the app but I think this is fine at this point :)